src/actions/makeArchive.ts

```diff
@@ -0,0 +1,39 @@
+import { withActionOutput } from '../utils';
+
+import WFArchiveFormat from '../interfaces/WF/WFArchiveFormat';
+import WFSerialization from '../interfaces/WF/WFSerialization';
+import WFWorkflowAction from '../interfaces/WF/WFWorkflowAction';
+
+/**
+ * Make Archive Action. Makes an archive out of the files passed as input.
+ *
+ *
+ * ```js
+ * makeArchive({
+ *   format: '.zip',
+ *   name: 'zipName',
+ * });
+ * ```
+ */
+const makeArchive = (
+  options: {
+    /** The operation to perform */
+    format?: WFArchiveFormat,
+    name?: WFSerialization | string,
```

Comment:
Great pull request! Could you add a comment for the name parameter too?

Reply:
@gcordalis, the Travis error is about a trailing whitespace after the comment that you added. Make sure there aren’t spaces after lines and check whether that line has some spaces at the end.

As a side note: Atom automatically removes trailing whitespaces on save 🙂

Reply:
Yeah, I use VS Code when on desktop/laptop but I do a bit of work on an iPad. I haven't got a good workflow for this on here yet. Need to get that sorted :)